index.js

```diff
@@ -75,8 +76,8 @@ const VERSION = "3.35.38", CELL_VOLUME = 16 * 16 * 16;
 const DEG2RAD = Math.PI / 180, RAD2DEG = 180 / Math.PI;
 
 function translateText(text) {
-	for (const [code, color] of Object.entries(COLOR_CODES)) text = text.replaceAll(code, color);
-	return text;
+	for (const [code, color] of Object.entries(COLOR_CODES)) new_text = text.replaceAll(code, color);
+	return new_text;
```

Comment:
```suggestion
	for (const [code, color] of Object.entries(COLOR_CODES)) text = text.replaceAll(code, color);
	return text;
```
why are you changing random stuff lol